### ISSUE TITLE
Unable to configure Standard Notes to use my self-hosted server. Instructions don't match the interface

### ISSUE THREAD

**Issue by motoridersd** [standardnotes/server#1063]
The instructions say:

"In the account menu, choose Advanced options. Under "Sync Server", select Custom, then enter the address of your new server."

But I don't see this in the Linux, Android or Web App (which I am also self-hosting).

Does this require a subscription? Is the web app supposed to have this option to point it to the self-hosted server? I would have expected this to work automatically when running on the same server.

**Comment by stevesbrain** [comment]
So when logging in, you don't see the "advanced options" bit here?
![image](https://github.com/user-attachments/assets/1353e322-0a70-4b8c-ae10-eb693db6d755)

**Comment by motoridersd** [comment]
I do see it now, but this is only when logging in. The instructions didn't mention that, so I had missed it. Maybe 

"In the account menu, choose Advanced options. Under "Sync Server", select Custom, then enter the address of your new server."

should be

"When logging in, choose Advanced options. Under "Sync Server", select Custom, then enter the address of your new server."

**Comment by stevesbrain** [comment]
No worries! This is also there when creating an account, so perhaps that is why they use more generic wording?

**Comment by motoridersd** [comment]
Yeah it could be, still not very clear that this is when logging in/creating account, but hopefully having this issue here will help others that might have a similar question.
